server/scripts/load-seed-data.js

```diff
@@ -0,0 +1,27 @@
+const pg = require('pg');
+const Client = pg.Client;
+const databaseUrl = 'postgres://localhost:5432/banana';
+const campgrounds = require('./campgrounds.json');
+
+const client = new Client(databaseUrl);
+
+client.connect()
+  .then(() => {
+    // "Promise all" does a parallel execution of async tasks
+    return Promise.all(
+      campgrounds.map(campground => {
+        return client.query(`
+          INSERT INTO campgrounds (name, forest, season, sites)
+          VALUES ($1, $2, $3, $4);
+        `,
+        [campground.name, campground.forest, campground.season]);
```

Comment:
```suggestion
        [campground.name, campground.forest, campground.season, campground.sites]);
```
Missing 4th column required on line 15. This prevented the load seed data script from working properly.